packages/shared-ux/chrome/navigation/src/ui/components/navigation_ui.tsx

```diff
@@ -24,11 +24,13 @@ export const NavigationUI: FC<Props> = ({ children, unstyled, footerChildren, da
         <EuiFlexGroup
           direction="column"
           gutterSize="none"
-          style={{ overflowY: 'auto' }}
+          style={{ height: `calc(100% - 48px)` }}
```

Comment:
Where does the `48px` value comes from? Is there any chance to do one of (or all):

- Expose it as a constant?
- Import the constant from the component that has that specific height?
- Dynamically read it from the DOM element itself using `getBoundingClientRect()` ?